packages/terra-demographics-banner/src/selectable-demographics-banner/_BackgroundTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes that are rendered within background tile.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: undefined,
+};
+
+const BackgroundTile = ({
+  children,
+  ...customProps
+}) => {
+  const backgroundTileClassNames = cx([
+    { deceased: customProps.deceasedDate },
```

Comment:
First class keys should not be keyed from customProps

Reply:
removed custom props and added first class prop [here](https://github.com/cerner/terra-core/pull/2856/commits/29442b337f5a3a0d8f21da8778dc7e6392406055#diff-9fd0ae0d0298ecb1d100f96d13178ad4)